Source/Scene/Cesium3DTileset.js

```diff
@@ -131,6 +134,31 @@ define([
         this._refineToVisible = defaultValue(options.refineToVisible, false);
 
         /**
+         * Whether the tileset should should refine based on a dynamic screen space error. Tiles that are further
+         * away will be rendered will lower detail that closer tiles. This improves performance by rendering fewer
+         * tiles and making less requests, but may result in a slight drop in visual quality for tiles in the distance.
+         *
+         * @see Fog
+         */
+        this.useDynamicScreenSpaceError = defaultValue(options.useDynamicScreenSpaceError, false);
```

Comment:
I think defaulting to false is safer for now. I've tried on a few datasets and having it on isn't too different but technically it does break the concept of a constant screen space error and we don't actually visualize it with fog.